README.md

```diff
@@ -53,7 +34,15 @@ make init
 
 | Project                                                      | Description                                                  |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| [Core extensions](https://github.com/adobe/aepsdk-core-android)                      | The Mobile Core represents the foundation of the Adobe Experience Platform mobile SDK.               |
+| [Edge Network](https://github.com/adobe/aepsdk-edge-android)                      | The Edge Network extension allows you to send data to the Adobe Edge Network from a mobile application.               |
+| [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) | The Identity extension enable identity management from your mobile app when using the Edge Network extension. |
+| [Assurance](https://github.com/adobe/aepsdk-assurance-android)                      | The Assurance extension enables validation workflows for your SDK implementation.              |
+| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDKs.                 |
```

Comment:
Tim pointed out avoiding usage of the AEP acronym, so edgeIdentity changed to Adobe Experience Platform sample app for Android